docs/docker/redirects.conf

```
@@ -116,6 +116,10 @@ rewrite ^/docs/vue3-simple-example/?$ /docs/javascript-data-grid/vue3-basic-exam
 rewrite ^/docs/vue3-setting-up-a-language/?$ /docs/javascript-data-grid/vue3-setting-up-a-translation/ permanent;
 rewrite ^/docs/row-sorting/?$ /docs/$framework/rows-sorting/ permanent;
 rewrite ^/docs/column-sorting/?$ /docs/$framework/rows-sorting/ permanent;
+rewrite ^/docs/javascript-data-grid/row-sorting/?$ /docs/javascript-data-grid/rows-sorting/ permanent;
+rewrite ^/docs/react-data-grid/row-sorting/?$ /docs/react-data-grid/rows-sorting/ permanent;
+rewrite ^/docs/javascript-data-grid/column-sorting/?$ /docs/javascript-data-grid/rows-sorting/ permanent;
+rewrite ^/docs/react-data-grid/column-sorting/?$ /docs/react-data-grid/rows-sorting/ permanent;
```

I guess Nginx does not support variable injecting in the input URL. I suggest rewriting the rules to something like this:
```suggestion
rewrite ^/docs/(javascript|react)-data-grid/row-sorting/?$ /docs/$framework/rows-sorting/ permanent;
rewrite ^/docs/(javascript|react)-data-grid/column-sorting/?$ /docs/$framework/rows-sorting/ permanent;
```